test/e2e/deployment.go

```diff
@@ -199,11 +204,11 @@ func testRollingUpdateDeploymentEvents(f *Framework) {
 	}
 	rcName := "nginx-controller"
 	replicas := 1
-	_, err := c.ReplicationControllers(ns).Create(newRC(rcName, replicas, rcPodLabels, "nginx", "nginx"))
+	_, err := clientset.Legacy().ReplicationControllers(ns).Create(newRC(rcName, replicas, rcPodLabels, "nginx", "nginx"))
```

Comment:
Hm... maybe we should call it "Core" instead of "Legacy". Should be an easy find and replace later?